src/main/java/hw_6/Resident.java

```diff
@@ -0,0 +1,26 @@
+package hw_6;
+
+public class Resident {
+    public String name;
+    public String surname;
+
+    public static int age;
+    public static String city;
```

Comment:
public access modifiers is it a good approach?

Reply:
changed